packages/node/test/stacktrace.test.ts

```diff
@@ -1,16 +1,4 @@
-/**
- * stack-trace - Parses node.js stack traces
- *
- * These tests were originally forked to fix this issue:
- * https://github.com/felixge/node-stack-trace/issues/31
```

Comment:
Maybe we should still leave a call out to the repo - even if the tests have transformed quite a bit since the fork (esp since `/Users/felix/code/node-fast-or-slow/lib/test.js` is scattered around everywhere)